galaxy_importer/loaders.py

```diff
@@ -165,9 +167,31 @@ def load(self):
     def _make_name(self):
         return os.path.basename(self.rel_path)
 
-    def _lint(self):
+    def _lint_role(self, path):
         self.log.info('Linting role via ansible-lint')
-        pass
+        cmd = [
+            'ansible-lint', path,
+            '-p',
+            '-x', 'metadata',
+        ]
+        self.log.debug('CMD: ' + ' '.join(cmd))
+        proc = Popen(
+            cmd,
+            cwd=self.root,
+            encoding='utf-8',
+            stdout=PIPE,
+        )
+
+        for line in proc.stdout:
+            line_list = line.split(' ')
+            rel_path = os.path.join(*Path(line_list[0]).parts[-3:])
```

Comment:
What is the `-3` magic number?

Reply:
This makes linter message have a shortened relative path, added a code comment note for this